### PR TITLE
[CP] Added show all in search form

### PR DESCRIPTION
**Pull request by thomasbisignani**

### CODE REVIEW

File: templates/search/_filters_radius.html.twig [hunk 1]
```diff
@@ -0,0 +1,11 @@
+<div>
+    dans un rayon de
+    {# this widget will be embedded in the ribbon text #}
+    <select name="{{ constant('PARAMETER_RADIUS', search) }}">
+        {% for option in constant('RADII', search) %}
+            {% if option != 0 %}
```
Comment (HeahDude) [inline templates/search/_filters_radius.html.twig]: The `if` can be put after the `for`:
```twig
{% for option in constant('RADII', search) if 0 != option %}
```

File: templates/search/_filters_radius_citizen_projects.html.twig [hunk 2]
```diff
@@ -0,0 +1,9 @@
+<div>
+    <select name="{{ constant('PARAMETER_RADIUS', search) }}">
+        {% for option in constant('RADII', search) %}
```
Comment (HeahDude) [inline templates/search/_filters_radius_citizen_projects.html.twig]: `0` is allowed here?

Comment (thomasbisignani) [reply templates/search/_filters_radius_citizen_projects.html.twig]: Yes. For show all Citizen Projects, the value is `0`.

File: templates/search/_filters_radius.html.twig (hunk 1 above)
Comment (thomasbisignani) [reply templates/search/_filters_radius.html.twig]: Nice catch, thx !

File: src/Search/SearchParametersFilter.php [hunk 3]
```diff
@@ -40,6 +40,7 @@ class SearchParametersFilter
         self::TYPE_CITIZEN_ACTIONS,
     ];
 
+    const RADIUS_ALL = 0;
```
Comment (blazarecki) [inline src/Search/SearchParametersFilter.php]: What about the `RADIUS_NONE = -1` ?

Comment (HeahDude) [reply src/Search/SearchParametersFilter.php]: "None" and "all" are the same in this context, I prefer "ALL" though.

File: src/Search/SearchParametersFilter.php [hunk 4]
```diff
@@ -88,7 +90,13 @@ public function handleRequest(Request $request): self
     {
         $this->setQuery((string) $request->query->get(self::PARAMETER_QUERY));
         $this->setType($request->query->get(self::PARAMETER_TYPE, ''));
-        $this->setRadius($request->query->getInt(self::PARAMETER_RADIUS));
+
+        $radius = null !== $request->query->get(self::PARAMETER_RADIUS) ?
```
Comment (HeahDude) [inline src/Search/SearchParametersFilter.php]: Could use `has` here, or store the result. But I would go for:
```php
$radius = $request->query->getInt(self::PARAMETER_RADIUS, self::DEFAULT_RADIUS);
```

File: src/Search/SearchParametersFilter.php (hunk 3 above)
Comment (blazarecki) [reply src/Search/SearchParametersFilter.php]: I don't think so, -1 is for user who don't want to saw citizen project. ping @thomasbisignani ?

Comment (thomasbisignani) [reply src/Search/SearchParametersFilter.php]: Ok for `-1` ! Thx.

**Pull request merged**